codechecker_common/report.py

```diff
@@ -251,7 +251,7 @@ def get_report_path_hash(report, files):
         col = str(event['location']['col']) if 'location' in event else 0
 
         report_path_hash += line + '|' + col + '|' + event['message'] + \
-            file_name
+            file_name + report.check_name
```

Comment:
Do we want to add the checker name for each event in the path? Is it not enough to add it only once to the hash content?

Please extend the function documentation about the change.

Reply:
Thanks for the good catch, it was not intentional. I didn't notice the loop few lines above.